css/style.css

```diff
@@ -25,6 +25,20 @@
 	padding: 10px;
 	margin-top: 0px;
 }
+.dragDropDisplay {
+  background: linear-gradient(to left, rgba(0, 0, 0, 0.9), rgba(0, 0, 0, 0.2)), url('/images/grass_block.png') top center repeat-x;
+  z-index: 1000;
+  font-family: Minecraft, Verdana, sans-serif;
+}
+.dragDropButton {
```

Comment:
Allow cursor movement to pass through to the dragDropDisplay for catching events.
```suggestion
.dragDropButton {
  pointer-events: none;
```